src/components/register/FirstStep.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div class="first-step-container">
+        <h1 class="headline">Ahoy You! <br> <span class="headline--accent">Care to register?</span></h1>
+        <form-card @submit.prevent.native="formSubmit">
+            <template #form-controls>
+                <div class="form-field">
+                    <base-input v-model="email" @blur="validateEmail" inputLabel="email" :isInvalid="isEmailError" placeholder="Enter your email"/>
+                </div>
+                <div class="form-field">
+                    <password-input v-model="password" :isFormSubmitted="isFormSubmitted" :validate="true" @validate="setPasswordValidity" />
+                </div>
+            </template>
+            <template #form-actions>
+                <base-button :block="true" type="button">
```

Comment:
tip: if a component has a prop declared as `Boolean` type (with no default value or `default: true`) you can use it like this:

when `:block="true"`
```html
<base-button type="button" block>
```
otherwise
```html
<base-button type="button">
```

Reply:
that's satisfying 😌